app.js

```diff
@@ -10,9 +10,9 @@ const dbConfig = require('./database/db');
 var cookieparser =require('cookie-parser');
 const csurf = require('csurf');
 var multer = require('multer');
-var upload = multer({
-  limits: { fileSize: 1024*1024 }
-});
+var upload = multer({ // creating multer
+  limits: { fileSize: 1024*1024*2 }
```

Comment:
#### Automated PR Comment From coverity
<!-- 32fd1203d5844cffcb40530566b080fe -->
<!-- Comment managed by Git Commenter, do not modify! -->
### Coverity Issue - Unrestricted number of uploaded file/non-file fields
_High CWE-400_
The `multer` module does not restrict the number of file/non-file fields in a single request. Thus, a denial-of-service attack is possible, because an attacker may submit a very large number of file/non-file fields in one request.
### How to fix
Limit the number of file fields in a request by setting the `files` property and the number of non-file fields by setting the `fields` property in the `limits` object of the `multer` options.